src/GraphODataTemplateWriter/CodeHelpers/TypeScript/TypeHelperTypeScript.cs

```diff
@@ -32,9 +32,11 @@ public static string GetTypeString(this OdcmProperty prop)
                 case "Int64":
                 case "Double":
                 case "Single":
-                case "Binary": // let binary: number = 0b1010;
                     typeStr = "number";
                     break;
+                case "Binary": // let binary: number = 0b1010;
```

Comment:
Could you please update the comments as well to reflect the change?

Reply:
Done